maro/simulator/scenarios/finance/README.md

```diff
@@ -0,0 +1,149 @@
+# Finance scenario in MARO
+
+## Introduction
+
+The Finance scenario simulates securities trading in financial markets.
```

Comment:
securities -> stock?

or provides examples, such as stock, foreign exchange.

And add currently, we only support stock.